hawc/apps/animalv2/models.py

```diff
@@ -0,0 +1,576 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from ..assessment.models import BaseEndpoint, DSSTox
+from ..vocab.models import Term
+from . import constants, managers
+
+"""
+Generate a GraphViz dot file of this model:
+
+    manage.py graph_models animalv2 > /path/to/file_to_create.dot
+
+(see https://django-extensions.readthedocs.io/en/latest/graph_models.html for details)
+
+Make an svg/png out of it (assuming you've installed graphviz):
+
+    dot -Tsvg /path/to/previously_generated_file.dot > converted.svg
+"""
+
+
+class Experiment(models.Model):
+    objects = managers.ExperimentManager()
+
+    # TODO - what does this do?!?!
```

Comment:
it allows those fields to be used in bulk_cleanup methods - ok to remove for now unless it throws an exception somewhere if it's not defined